src/adapters/bitbucket.js

```diff
@@ -0,0 +1,255 @@
+const BB_RESERVED_USER_NAMES = [
+  'account', 'dashboard', 'integrations', 'product',
+  'repo', 'snippets', 'support', 'whats-new'
+]
+const BB_RESERVED_REPO_NAMES = []
+const BB_RESERVED_TYPES = ['raw']
+const BB_404_SEL = '#error.404'
+const BB_PJAX_CONTAINER_SEL = '#source-container'
+
+class Bitbucket extends Adapter {
+
+  constructor() {
+    super(['jquery.pjax.js'])
+
+    $.pjax.defaults.timeout = 0 // no timeout
+    $(document)
+      .on('pjax:send', () => $(document).trigger(EVENT.REQ_START))
+      .on('pjax:end', () => $(document).trigger(EVENT.REQ_END))
+  }
+
+  // @override
+  init($sidebar) {
```

Comment:
This method & selectFile() duplicate with github adapter a lot, maybe add a PjaxAdapter to keep the share code?